[2022-12-11 19:11:27,613] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.wait_for_file manual__2022-12-11T19:11:19.366812+00:00 [queued]>
[2022-12-11 19:11:27,627] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.wait_for_file manual__2022-12-11T19:11:19.366812+00:00 [queued]>
[2022-12-11 19:11:27,628] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:11:27,629] {taskinstance.py:1357} INFO - Starting attempt 1 of 3
[2022-12-11 19:11:27,630] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:11:27,650] {taskinstance.py:1377} INFO - Executing <Task(PythonSensor): wait_for_file> on 2022-12-11 19:11:19.366812+00:00
[2022-12-11 19:11:27,655] {standard_task_runner.py:52} INFO - Started process 4663 to run task
[2022-12-11 19:11:27,659] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'wait_for_file', 'manual__2022-12-11T19:11:19.366812+00:00', '--job-id', '1225', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpy3lwqtjv', '--error-file', '/tmp/tmp1zbb84xz']
[2022-12-11 19:11:27,664] {standard_task_runner.py:80} INFO - Job 1225: Subtask wait_for_file
[2022-12-11 19:11:27,740] {task_command.py:369} INFO - Running <TaskInstance: first_dag.wait_for_file manual__2022-12-11T19:11:19.366812+00:00 [running]> on host 7e39590a7807
[2022-12-11 19:11:27,842] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=wait_for_file
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T19:11:19.366812+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T19:11:19.366812+00:00
[2022-12-11 19:11:27,846] {python.py:66} INFO - Poking callable: <function _wait_for_file at 0x7f6cec9b1cb0>
[2022-12-11 19:11:27,861] {logging_mixin.py:115} INFO - _wait_for_file True
[2022-12-11 19:11:27,863] {logging_mixin.py:115} INFO - total attempt 0
[2022-12-11 19:11:47,879] {python.py:66} INFO - Poking callable: <function _wait_for_file at 0x7f6cec9b1cb0>
[2022-12-11 19:11:47,928] {logging_mixin.py:115} INFO - _wait_for_file True
[2022-12-11 19:11:47,931] {logging_mixin.py:115} INFO - total attempt 0
[2022-12-11 19:12:07,926] {python.py:66} INFO - Poking callable: <function _wait_for_file at 0x7f6cec9b1cb0>
[2022-12-11 19:12:07,957] {logging_mixin.py:115} INFO - _wait_for_file True
[2022-12-11 19:12:07,958] {logging_mixin.py:115} INFO - total attempt 0
[2022-12-11 19:12:07,960] {base.py:301} INFO - Success criteria met. Exiting.
[2022-12-11 19:12:07,995] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=wait_for_file, execution_date=20221211T191119, start_date=20221211T191127, end_date=20221211T191207
[2022-12-11 19:12:08,037] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 19:12:08,134] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
